serverless_template.yml

```diff
@@ -84,17 +86,17 @@ functions:
     events:
       - http:
           method: post
-          path: images
           cors: true
+          path: /
```

Comment:
In order to upload an image, you just `POST` to `images.image-service-<stage>.pinster.io` which I think makes a bit more sense hopefully.

Reply:
👍